test/junit/scala/math/BigDecimalTest.scala

```diff
@@ -287,4 +287,29 @@ class BigDecimalTest {
   def testIsComparable(): Unit = {
     assert(BigDecimal(0.1).isInstanceOf[java.lang.Comparable[_]])
   }
+
+  
+
+  @Test
+  def testBigDecimalSumInList(): Unit = {
+
+    val bds = List(
+      scala.math.BigDecimal("1000000000000000000000000.1", MC.UNLIMITED),
```

Comment:
these don't have to be prefixed with `scala.math.`. I assume they were in the issue comment because it was not written as part of this test file.

```suggestion
      BigDecimal("1000000000000000000000000.1", MC.UNLIMITED),
```